accelerator/config.c

```diff
@@ -70,6 +72,13 @@ status_t cli_core_set(char* conf_file, ta_config_t* const ta_conf, iota_config_t
       cache->port = atoi(value);
       break;
 
+    // DB configuration
+    case DB_ENABLE_CLI:
+      db_service->enabled = (toupper(value[0]) == 'T');
+      break;
+    case DB_HOST_CLI:
+      db_service->host = value;
```

Comment:
Is it safe to use assignment rather than `strdup` here?

Reply:
I think the assignment is not safe.
I will use `strdup `here and create a new issue to revise all other configuration assignments through CLI.

Reply:
In case that using `strdup` here, I need to use `strdup` when giving a default value either.
That is revise `db_service->host = DB_HOST;`
Otherwise, we won't know `db_service->host` was malloc or not when performing `free()`.